src/xr.js

```diff
@@ -75,6 +75,31 @@ function promise(args, fn) {
   )(fn);
 }
 
+function getURI(opts) {
+	let uri = opts.url;
+	let subPath = opts.subPath
+	if(!uri|| subPath.indexOf("://") !== -1) {
```

Comment:
A substantial number of users are likely to use 'protocol relative' urls - which will not contain `://`
This probably won't break existing implementations (since folks are unlikely to have specified a default `url` as the base url) - however it will prevent the usage of the override should a base url be specified during initialization.